prometheus_aioexporter/script.py

```diff
@@ -3,6 +3,8 @@
 import argparse
 from collections.abc import Iterable
 import logging
+import ssl
+from ssl import SSLContext
```

Comment:
Given yoou're importing `ssl`, you can just use `ssl.SSLContext` below and drop this extra import

Reply:
nice catch - removed!